src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/WindowsFormsSynchronizationContextTests.cs

```diff
@@ -34,6 +34,7 @@ public void WindowsFormsSynchronizationContext_CreateCopy_Invoke_Success()
             Assert.Equal(2, callCount);
         }
 
+        /* https://github.com/dotnet/winforms/issues/3297
         [WinFormsFact]
```

Comment:
Instead of commenting out, please skip:
```suggestion
        [WinFormsFact(Skip = "WindowsFormsSynchronizationContext disposed of too early. See: https://github.com/dotnet/winforms/issues/3297")]
        [ActiveIssue("https://github.com/dotnet/winforms/issues/3297")]
```

Reply:
I' happy to try again to see if it works in CI, but locally skipping the tests was leading to errors in test runs. The other two are WinFormsTheory and xunit was complaining the argument count wasn't matching the number of passed arguments. Weird behavior for a skipped test.

Reply:
I'll pull down tomorrow, and have a look